objects/src/lib.rs

```diff
@@ -60,6 +60,65 @@ pub mod utils {
             Digest::new([Felt::new($a), Felt::new($b), Felt::new($c), Felt::new($d)])
         };
     }
+
+    /// Construct a new `Digest` from long hex value.
+    #[macro_export]
+    macro_rules! digest_from_hex {
```

Comment:
Do we have any other conversion macros? If not, I'd probably use just `digest!`.

Reply:
I left the macro we were using previously, which has `digest` name, but looks like we don't use it anywhere else, so I can remove it and rename the new macro to `digest`.